force-app/main/default/lwc/hot_recordFiles/hot_recordFiles.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>false</isExposed>
+</LightningComponentBundle>
```

Comment:
Kan denne generaliseres og flyttes til crm-community-base, som en egen komponent?
Trenger nok da også å opprette ny apex klasse hvor metoden ligger.
Vi kan også utvide komponenten til å kunne ta inn en liste med ContentDocuments, istedenfor record Id, om man ikke ønsker å vise ALLE filene på record. Utvikler kan da lage en egen apex kontroller for å hente ut, evt sortere i sitt js.

Reply:
Kan denne også utvides til støtte av sletting av filer, som vi har i request form? Kanskje en egen parameter per element isDeletable?
@api isAllDeleteable kan også benyttes, om utvikler ønsker samme funksjonalitet for alle vedleggene i lista.

Reply:
@vmjelde Ja, men sletting var vel ikke prio atm?

Reply:
Tenker det er fase 2 ja.
https://jira.adeo.no/browse/TOLK-1667
Se kommentar fra Svenn Terje